src/net_processing.h

```diff
@@ -75,7 +75,8 @@ class PeerLogicValidation final : public CValidationInterface, public NetEventsI
     void CheckForStaleTipAndEvictPeers(const Consensus::Params &consensusParams);
     /** If we have extra outbound peers, try to disconnect the one with the oldest block announcement */
     void EvictExtraOutboundPeers(int64_t time_in_seconds) EXCLUSIVE_LOCKS_REQUIRED(cs_main);
-
+    /** Retrieve unbroadcast transactions from the mempool and possibly send to pnode */
+    void QueueUnbroadcastTxs(CNode* pnode) override;
```

Comment:
in commit 2f3f449:

Can be const?

```suggestion
    void QueueUnbroadcastTxs(CNode& node) const override;
```

Reply:
Method no longer exists but made `ReattemptInitialBroadcast` const.